pkgs/by-name/de/deepsource/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "deepsource";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner = "DeepSourceCorp";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-6uNb4cQVerrlW/eUkjmlO1i1YKYX3qaVdo0i5cczt+I=";
+  };
+
+  doCheck = false;
```

Comment:
Indicate why in a comment on top of or behind this line.

Reply:
updated